### DISCUSSION TITLE
Any idea, when to release Sloeber 4.4.1?

### DISCUSSION THREAD

**Discussion by ArminJo** [Sloeber/arduino-eclipse-plugin#1496] [General]
Hi Jantje,
I will hold a talk "Arduino erfolgreich programmieren mit Sloeber/Eclipse" at the Hannover Maker faire next weekend https://maker-faire.de/hannover/vortraege-und-workshops/.
Now I wonder, when you plan to release the 4.4.1 version I will present. (I dont like to have outdated documentations/presentations).
So is there a date, I can communicate, when the 4.4.1 release is planned to be released?

Thanks for your amazing work!
Armin

**Comment by jantje** [comment]
I just released 4.4.1 here on github.
I will probably do the changes for the website tomorrow. Not sure when they will be pushed to baeyens.it

**Reply by ArminJo** [reply]
Thanks for the Info!

**Reply by ArminJo** [reply]
First test of replacing updated 4.4.1 version with fresh release version were successful 😀 👍

**Comment by jantje** [comment]
The release is exactly the same as this one https://github.com/Sloeber/arduino-eclipse-plugin/actions/runs/2723411208

**Comment by jantje** [comment]
The deed has been done. 😎
